docs/DeveloperGuide.md

```diff
@@ -293,23 +293,35 @@ The following activity diagram summarizes what happens when a user enters an `ed
 ### MarkRoomsUnclean Command
 
 #### Implementation
-* The `markRoomsUnclean` command edits all the guests in GuestBook and changes their isRoomClean statuses to "no". It takes in no additional inputs or fields.
+* The `markroomsunclean` command edits all the guests in GuestBook and changes their isRoomClean statuses to "no". It takes in no additional inputs or fields.
 
-The following activity diagram summarizes what happens when a user enters a `markRoomsUnclean` command.
+The following activity diagram summarizes what happens when a user enters a `markroomsunclean` command.
```

Comment:
Should we use summarises instead for consistency?

Reply:
Fixed!